From 49ea7ccc53a99ba8be1a632319122ecdae1fe4a1 Mon Sep 17 00:00:00 2001
From: Jacob Pfeifer <jacob@pfeifer.dev>
Date: Fri, 17 May 2024 21:50:36 -0400
Subject: [PATCH] ford steering hack

---
 selfdrive/car/ford/carcontroller.py | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/selfdrive/car/ford/carcontroller.py b/selfdrive/car/ford/carcontroller.py
index 47082fb56..ea1876c9e 100644
--- a/selfdrive/car/ford/carcontroller.py
+++ b/selfdrive/car/ford/carcontroller.py
@@ -65,6 +65,13 @@ class CarController(CarControllerBase):
       if CC.latActive:
         # apply rate limits, curvature error limit, and clip to signal range
         current_curvature = -CS.out.yawRate / max(CS.out.vEgoRaw, 0.1)
+        # PFEIFER - FSH {{
+        # Ignore limits while overriding, this prevents pull when releasing the wheel. This will cause messages to be
+        # blocked by panda safety, usually while the driver is overriding and limited to at most 1 message while the
+        # driver is not overriding.
+        if CS.out.steeringPressed:
+          self.apply_curvature_last = actuators.curvature
+        # }} PFEIFER - FSH
         apply_curvature = apply_ford_curvature_limits(actuators.curvature, self.apply_curvature_last, current_curvature, CS.out.vEgoRaw)
       else:
         apply_curvature = 0.
-- 
2.39.2

